code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm

```diff
@@ -493,6 +493,7 @@
 			sister.remove_filter("summonoutline")
 		return fail_activate()
 
+	allxenos = X.hive.get_all_xenos() //refresh the list to account for any changes during the channel
```

Comment:
Will this really resolve the issue? There is no check for dead, or whatever within `/proc/get_all_xenos`

Reply:
that procs uses xenos_by_typepath which is updated when xenos die etc, otherwise shrike/queen evo checks would always fail as they use the same list.